pkg/sypgp/sypgp.go

```diff
@@ -587,9 +594,12 @@ func SelectPubKey(el openpgp.EntityList) (*openpgp.Entity, error) {
 
 // SelectPrivKey prints a secret key list to user and returns the choice
 func SelectPrivKey(el openpgp.EntityList) (*openpgp.Entity, error) {
+	if len(el) <= 0 {
```

Comment:
Sorry to bother, but length cannot be less than 0 😃

So the common way to check slices regardless of nil is `if len(slice) == 0`. Just to look Go native. Otherwise lgtm!

Reply:
Okay, I will fix in a bit :+1:

But, is it possible for it to return `-1`? as a error maybe? I guess Go does not do that...

Reply:
no, len never returns negative, it is in go spec